lib/gke-overrides.libsonnet

```diff
@@ -28,73 +42,208 @@ local ignore_dashboards = [
   'scheduler.json',
 ];
 
+// Define a mapping of alert/recordname to expression.
+// Overrides the expr field for the specifeid record name.
+local expr_overrides = {};
+
+// Define a mapping of alertname to label page:false
+// It will add a label `page:false` else it will add a `page:true`
+local page_false_critical = [
+  'AlertmanagerConfigInconsistent',
+  'ContainerCombinedIoHighOverTime',
+  'ClusterLowMemoryAvailable',
+  'ElasticsearchClusterHealthYELLOW',
+  'ElasticsearchHeapTooHigh',
+  'ElasticsearchLowDiskFree',
+  'HAProxyFrontendSessionUsage',
+  'HAProxyBackendResponseErrors',
```

Comment:
Couldn't this basically end up being the same as SiteIsDown (at least from the user-viewpoint). 

I guess it depends how many errors, but if it's 100% over time, that should page right?

Reply:
I don't know as of now what the metrics represent.

We have another one that checks for percentage of 5xx over total number of errors, that one still page .

this one is disabled from paging so that we can see what / when it triggers.

the SiteIsDown is not really the same, the blackbox rules are **harder** , they will start to error when things are really bad. This one will pick a smaller percentage of 5XX errors

Reply:
Alright, makes sense as there's the blackbox and additional alerts already covering this